<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>scissor_diag</class>
 <widget class="QDialog" name="scissor_diag">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>225</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Work Mode:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Debug Mode:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sc_image_only">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Image Only</string>
   </property>
  </widget>
  <widget class="QPushButton" name="image_contour">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Image with 
 Contour</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>241</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>110</y>
     <width>251</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>250</y>
     <width>251</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="sc_OK">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>420</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sc_cost_graph">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>160</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Cost Graph</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sc_pixel_node">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Pixel Nodes</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sc_min_path">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>210</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Minimum Path</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sc_path_tree">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Path Tree</string>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>400</y>
     <width>251</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Extra</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sc_brush">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Brush ON</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sc_snap_on">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>300</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Snap Seed OFF</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sc_cost_function">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Cost function(Modified)</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>sc_OK</sender>
   <signal>clicked()</signal>
   <receiver>scissor_diag</receiver>
   <slot>sc_OK()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>115</x>
     <y>520</y>
    </hint>
    <hint type="destinationlabel">
     <x>125</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>image_contour</sender>
   <signal>clicked()</signal>
   <receiver>scissor_diag</receiver>
   <slot>sc_image_contour()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>125</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sc_image_only</sender>
   <signal>clicked()</signal>
   <receiver>scissor_diag</receiver>
   <slot>sc_image_only()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>55</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>125</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sc_pixel_node</sender>
   <signal>clicked()</signal>
   <receiver>scissor_diag</receiver>
   <slot>sc_pixel_node()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>55</x>
     <y>180</y>
    </hint>
    <hint type="destinationlabel">
     <x>125</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sc_cost_graph</sender>
   <signal>clicked()</signal>
   <receiver>scissor_diag</receiver>
   <slot>sc_cost_graph()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>180</y>
    </hint>
    <hint type="destinationlabel">
     <x>125</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sc_path_tree</sender>
   <signal>clicked()</signal>
   <receiver>scissor_diag</receiver>
   <slot>sc_path_tree()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>55</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>125</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sc_min_path</sender>
   <signal>clicked()</signal>
   <receiver>scissor_diag</receiver>
   <slot>sc_min_path()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>125</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sc_snap_on</sender>
   <signal>clicked()</signal>
   <receiver>scissor_diag</receiver>
   <slot>sc_snap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>320</y>
    </hint>
    <hint type="destinationlabel">
     <x>125</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sc_brush</sender>
   <signal>clicked()</signal>
   <receiver>scissor_diag</receiver>
   <slot>sc_brush()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>55</x>
     <y>320</y>
    </hint>
    <hint type="destinationlabel">
     <x>112</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sc_cost_function</sender>
   <signal>clicked()</signal>
   <receiver>scissor_diag</receiver>
   <slot>cs_cf()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>55</x>
     <y>380</y>
    </hint>
    <hint type="destinationlabel">
     <x>112</x>
     <y>280</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>sc_image_only()</slot>
  <slot>sc_image_contour()</slot>
  <slot>sc_pixel_node()</slot>
  <slot>sc_cost_graph()</slot>
  <slot>sc_path_tree()</slot>
  <slot>sc_min_path()</slot>
  <slot>sc_OK()</slot>
  <slot>sc_snap()</slot>
  <slot>sc_brush()</slot>
  <slot>cs_cf()</slot>
  <slot>sc_change_blur()</slot>
 </slots>
</ui>
